Marlin/src/gcode/gcode.cpp

```diff
@@ -93,6 +93,9 @@ int8_t GcodeSuite::get_target_extruder_from_command() {
     SERIAL_ECHOLNPAIR(" " MSG_INVALID_EXTRUDER " ", int(e));
     return -1;
   }
+  #if ENABLED(SINGLENOZZLE)
+    if(active_extruder<0) return 0;
+  #endif
```

Comment:
On single nozzle you should only have 0 or higher, if it was throwing a number less it sounds like a use before init

Reply:
this is not needed. this was only a test for me and had nothing to do with G60/G61